docs/guides/getting_started.md

```diff
@@ -47,46 +47,34 @@ Contact your babylon admin to get your **URI** service and your **userpass**.
 
 * If entries does not exist, you may need to create it (e.g for a newly deployed platform):
   We provide a tool for this, you can find it [here](https://github.com/Cosmo-Tech/backend-tf-state-to-vault)
-  This needs an initial Terraform depoyment as it uses the Terraform state to parse and populate the Vault.
-  Usefull informations can be found in the Readme of this repository !
+  This needs an initial Terraform deployment as it uses the Terraform state to parse and populate the Vault.
+  Useful information can be found in the Readme of this repository !
 
 ### Configuration
 
 At this point, you will need two variables to perform Babylon commands.
 
   - `context_id` : project name
   - `platform_id` : platform name
+  - `state_id`: state name
 
-For example, you can try this command,
-
-```bash
-babylon config display -c brewery -p dev
-```
-You will see the current configuration. For now, this configuration is empty.
-
-To retrieve `dev` configuration, you need perform this command.
-
-```bash
-babylon config init -c brewery -p dev
-```
-
-Try again the display command and verify the current configuration.
+`context_id` and `state_id` can be strings of your choice,
+but they cannot contain special characters. To initialize it, perform this command:
```

Comment:
`context_id` and `state_id` can be strings of your choice,
but they cannot contain special characters.  `platform_id` represents the ID of the platform, such as: dev, staging, etc.